apps/indexer/bin/comment-references-refresher-worker.ts

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env -S node --experimental-transform-types
+
+/**
+ * This script retries comment reference resolution for failed or partial results.
+ * It processes records from comment_reference_resolution_results table that have
+ * status 'failed' or 'partial' and are not too old (configurable via parameter).
+ */
```

Comment:
@normanzb please here a disclaimer that this script doesn't support multiple workers because it doesn't use locks.